src/classes/BGE_BatchGiftEntry_UTIL.cls

```diff
@@ -102,4 +102,40 @@ public with sharing class BGE_BatchGiftEntry_UTIL {
         };
     }
 
+    /*******************************************************************************************************
+    * @description Returns the source field's options formatted for lightning:select if the source field is a picklist
+    * or the target field's options formatted for lightning:select if the source field is a string and the target
+    * field is a picklist
+    * Returns an empty map if neither case applies
+    */
+    public static List<Map<String, String>> getPicklistOptions(String dataImportField){
+        List<Map<String, String>> options = new List<Map<String, String>>();
+
+        String dataImportObject = UTIL_Namespace.StrTokenNSPrefix('DataImport__c');
+
+        String rawSourceFieldType = UTIL_Describe.getFieldDisplaytype(dataImportObject, dataImportField.toLowerCase()).name().toLowerCase();
+
+        if (rawSourceFieldType == 'picklist') {
```

Comment:
I know that this code is just moving to better place (yay!) but is there a defined constant that we can use for 'picklist' and 'string' here instead of hard-coding the strings?